### ISSUE TITLE
The config command can't be ran

### ISSUE THREAD

**Issue by EmmaKnijn** [MCJack123/craftos2#280]
**Describe the bug**
Config can't be opened

**To Reproduce**
1. Open CraftOS-PC
2. Start typing config
3. See error

**Expected behavior**
Being able to run config

**Screenshots**
![image](https://user-images.githubusercontent.com/64010592/193452409-0c4b8c58-1180-44d8-9078-0b9c3e33ac41.png)
![image](https://user-images.githubusercontent.com/64010592/193452591-4bc39364-a96f-4d70-9e0a-a79a7c3f157f.png)
![image](https://user-images.githubusercontent.com/64010592/193452596-1592679e-fb36-47ee-9489-28441d05dba4.png)

**Environment:**
 - OS: Windows 10
 - OS Version: 21H2
 - CraftOS-PC Version: v2.7.1
 - Compiled from source?: No

The crashdump that it gave me after waiting on the error for a while
[CraftOS-PC.exe.21632.dmp.zip](https://github.com/MCJack123/craftos2/files/9692384/CraftOS-PC.exe.21632.dmp.zip)

**Comment by EmmaKnijn** [comment]
Duplicate of #273
